packages/@ama-sdk/showcase-sdk/tsconfig.build.json

```diff
@@ -24,24 +24,6 @@
     "composite": true,
     "declaration": true,
     "declarationMap": false,
-    "rootDir": "./src",
-    "baseUrl": ".",
```

Comment:
should we remove it from the generators as well ? https://github.com/AmadeusITGroup/otter/blob/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base/tsconfig.build.json#L26

Reply:
That's a good question.
What was the purpose of having this in the generators? Do people use those paths?
In the generators, by default, they do not cause any issue in the same way as they caused an issue for showcase-sdk, because the generated tsconfig does not extend anything (so it does not override any other paths defined somewhere else).
But people can easily have this problem in case they extend another tsconfig.json and want to define paths there.

Reply:
I have added a commit to remove paths from the tsconfig in the generator.